src/libraries/System.Net.Quic/src/System.Net.Quic.csproj

```diff
@@ -9,10 +9,22 @@
     <TargetPlatformIdentifier>$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)'))</TargetPlatformIdentifier>
     <DefineConstants Condition="'$(TargetPlatformIdentifier)' == 'FreeBSD' or '$(TargetPlatformIdentifier)' == 'OSX'">$(DefineConstants);SOCKADDR_HAS_LENGTH</DefineConstants>
     <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetPlatformIdentifier)' == ''">SR.SystemNetQuic_PlatformNotSupported</GeneratePlatformNotSupportedAssemblyMessage>
+    <ApiExclusionListPath Condition="'$(TargetPlatformIdentifier)' == ''">ExcludeApiList.PNSE.txt</ApiExclusionListPath>
   </PropertyGroup>
   <!-- Source files -->
   <ItemGroup Condition="'$(TargetPlatformIdentifier)' != ''">
-    <Compile Include="System\Net\Quic\*.cs" />
+    <Compile Include="System\Net\Quic\NetEventSource.Quic.cs" />
```

Comment:
You can use something like this:
```
<Compile Include="System\Net\Quic\*.cs" Exclude="QuicImplementationProviders.Unsupported.cs">
```
Instead of naming out all the files.
Up to you, I don't mind either way.

Reply:
I think we should enumerate all files here to be consistent with other project files. I kept the wildcards in the rest of the file since that part can still change quite often.